applications/softdevice/README.rst

```diff
@@ -0,0 +1,7 @@
+.. _softdevice_flasher_application:
+
+SoftDevice flasher application
+##############################
+
+This is a project used by sysbuild to sign the SoftDevice image automatically when ``west flash`` is invoked.
```

Comment:
Is it correct that this project signs the SoftDevice image? AFAIU, it only flashes the SoftDevice hex.